receptai.api/Interfaces/IRecipeRepository.cs

```diff
@@ -5,7 +5,7 @@ namespace receptai.api.Interfaces;
 
 public interface IRecipeRepository
 {
-    Task<List<Recipe>> GetAllAsync();
+    Task<List<Recipe>> GetAllAsync(int pageNumber = 1, int pageSize = int.MaxValue);
```

Comment:
Also - leaving as ALL entries is 100% wrong, maybe some sensible default like 10, 50, maybe 100 would make more sense.

Because now you are fetching (up to) 2,147,483,647 entries